package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "storybook-addon-vue-mdx",
-  "version": "0.1.5",
+  "version": "0.2.5",
```

```suggestion
  "version": "0.2.0",
```

Forgot about that: 0.2.0 would be more appropriate for a new major :)